perl/p5-graveyard/Portfile

```diff
@@ -18,16 +18,20 @@ proc unknown args {
         lappend args {*}$defaultBranches
     }
     set replaced_branches [lassign $args superport version_string]
-    lassign [split $version_string _] ver rev
+    lassign [split $version_string _] ver rev epo
```

Comment:
How does this work with the `@` sign?

Reply:
@mojca regarding your comments above: @ryandesign already updated the lists of ports with the final naming scheme, but the actual code to parse this is still what I initially suggested. I will change this accordingly and update this PR - for now it indeed does not work ;)